src/components/ClientMainParts/PreacherPart.vue

```diff
@@ -10,21 +10,17 @@
             </div>
         </div>
     </div>
-    <div v-else><CircleSpinner /></div>
 </template>
 
 <script>
-  import SectionTitle from "@/components/SectionTitle";
-  import { mapState } from 'vuex';
+import { mapState } from 'vuex';
+import SectionTitle from '@/components/SectionTitle';
 
-  export default {
-    name: "PreacherPart",
-    components: { SectionTitle },
-    computed: mapState({
-      loading: state => state.preachers.loading,
-      preacher: state => state.preachers.preacher,
-    }),
-  }
+export default {
+  name: 'PreacherPart',
+  components: { SectionTitle },
```

Comment:
wrong usage. vue consider to use kebab-case. You didn't rename usage of this component
right variant components: {
's-section-title': SectionTitle
}

 a good practice to use your project prefix, like 's' or 'sf'